requirements/env_climada.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
   - pyepsg>=0.4
   - pytables>=3.6
   - python=3.8
+  - pyxlsb=1.0.8  
```

Comment:
Yes I actually asked emanuel in the merge request about this. It is needed if we want to download the WIOT tables, otherwise we would need to store them in CLIMADA and they are big exceel sheets (something about 2400x2400 matrixes).

Reply:
Perfect, thanks for checking with Emanuel!

Reply:
Well let's see what he thinks..

Reply:
It seems it can be added